src/cdk/overlay/_overlay.import.scss

```diff
@@ -0,0 +1,8 @@
+@forward 'overlay' hide $dark-backdrop-background, $z-index-overlay, $z-index-overlay-backdrop,
```

Comment:
I just tried this locally and everything does work as expected, I just don't understand _why_ it works. We copy `_a11y.scss`, `_overlay.scss`, and `_text-field.scss` into the [root of the package](https://unpkg.com/browse/@angular/cdk@11.0.2/). Since the names of the mixins changed (e.g. `cdk-overlay` to `overlay`), I thought that the following would fail:
```scss
@import '~@angular/cdk/overlay';
@include cdk-overlay();
```

But it actually works just fine.

Reply:
That doesn't sound right, you might have something cached which is preventing it from failing. I'll look into copying the `.import` files too.